romanisim/image.py

```diff
@@ -289,18 +295,24 @@ def add_objects_to_image(image, objlist, xpos, ypos, psf,
             psf0 = psf.at_position(xpos[i], ypos[i])
         else:
             psf0 = psf
-        final = galsim.Convolve(profile * flux_to_counts_factor, psf0)
+        if isinstance(flux_to_counts_factor, list):
+            final = galsim.Convolve(profile * flux_to_counts_factor[i], psf0)
+        else:
+            final = galsim.Convolve(profile * flux_to_counts_factor, psf0)
```

Comment:
Minor stylistic thing, but my preference would be to define something like factor = flux_to_counts_factor[i] if isinstance(...) else flux_to_counts_factor and then to let the convolution like be galsim.Convolve(profile * factor, psf0), just aiming to avoid duplicating the convolution line.

Reply:
Implemented.